src/main/java/io/redisearch/client/Client.java

```diff
@@ -226,6 +226,71 @@ public boolean alterIndex(Schema.Field ...fields) {
         }
     }
 
+    /**
+     * Set runtime configuration option
+     *
+     * @param option the name of the configuration option
+     * @param value a value for the configuration option
+     * @return
+     */
+    @Override
+    public boolean setConfig(ConfigOption option, String value) {
```

Comment:
@dengliming thanks for the PR please review the PR I sent you with reduced memory allocations